app/models/role.rb

```diff
@@ -17,6 +18,14 @@ def self.add_admin(person)
     find_or_create_by(person_id: person.id, name: "admin")
   end
 
+  def self.moderator?(person)
+    exists?(person_id: person.id, name: "moderator")
+  end
```

Comment:
@jhass we also reflected some alternatives to the current design... we'll think it over again. I find it a little bit confusing to only mention moderators but include admins. But @Realtin also was pro including both roles somehow.